pkgs/applications/video/handbrake/default.nix

```diff
@@ -43,12 +45,13 @@ stdenv.mkDerivation rec {
     fribidi fontconfig freetype jansson zlib
     libass libiconv libsamplerate libxml2 bzip2
     libogg libopus libtheora libvorbis libdvdcss a52dec libmkv
-    lame ffmpeg libdvdread libdvdnav libbluray mp4v2 mpeg2dec x264 x265 libvpx
-  ] ++ (lib.optionals useGtk [
+    lame ffmpeg libdvdread libdvdnav libbluray mp4v2 mpeg2dec x264 x265 libvpx]
```

Comment:
We keep spaces around items in a list.

Reply:
Ok.
Not my style, but I would do in community style to become consistent, on the next time.
I learn.